client/src/Tasks/TaskRow.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Chip from '@material-ui/core/Chip';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import Panel from '@material-ui/core/ExpansionPanel';
+import PanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import PanelDetails from '@material-ui/core/ExpansionPanelDetails';
+
+import { useStyles } from './task.styles';
+import { Task } from '../common/types';
+import { FILTER, dictionary } from './tasks.config';
+import { Logs, useTaskRow } from './taskRow.hooks';
+
+interface Props {
+    task: Task;
+    routeId: number;
+    deleteTask: (id: number) => void;
+}
+
+const TaskRow: React.FC<Props> = ({ task, routeId, deleteTask }) => {
```

Comment:
you can unpack `task` here so you won't repeat yourself below when using `task.` in few places